src/components/[guild]/AddRewardButton/AddRewardButton.tsx

```diff
@@ -21,6 +21,7 @@ export type AddRewardForm = {
   requirements?: Requirement[]
   roleIds?: number[]
   visibility: Visibility
+  name?: string // Name for role, if new role is created with reward
```

Comment:
We could call this `roleName` so we wouldn't need to explain it in a comment, what do you think?